packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -555,12 +555,14 @@ def __init__(
         :param cross_period_persisted_keys: data keys that will be kept after a new period starts
         """
         AbciAppDB._check_data(setup_data)
-        self._setup_data = setup_data
-        self._cross_period_persisted_keys = cross_period_persisted_keys or []
+        self._setup_data = deepcopy(setup_data)
+        self._cross_period_persisted_keys = (
+            cross_period_persisted_keys.copy()
+            if cross_period_persisted_keys is not None
+            else []
+        )
         self._data: Dict[int, Dict[str, List[Any]]] = {
-            RESET_COUNT_START: deepcopy(
```

Comment:
Why no longer here?

Reply:
Because the `property` is returning a `deepcopy` now:

https://github.com/valory-xyz/open-autonomy/blob/e8f5b80b4c164af7dd663044ac8ff413d7715b49/packages/valory/skills/abstract_round_abci/base.py#L569-L577